--- src/unix/unix.mak~	2006-05-26 00:52:07.000000000 -0400
+++ src/unix/unix.mak	2011-02-03 03:02:00.000000000 -0500
@@ -366,7 +366,7 @@
 endif
 
 ifdef SOUND_ARTS_TEIRA
-SOUND_OBJS += $(DSP_DIR)/artssound.o
+# SOUND_OBJS += $(DSP_DIR)/artssound.o
 endif
 
 ifdef SOUND_ARTS_SMOTEK
@@ -457,13 +457,13 @@
 endif
 
 ifdef SOUND_ARTS_TEIRA
-CONFIG  += -DSYSDEP_DSP_ARTS_TEIRA `artsc-config --cflags`
-LIBS += `artsc-config --libs`
+# CONFIG  += -DSYSDEP_DSP_ARTS_TEIRA `artsc-config --cflags`
+# LIBS += `artsc-config --libs`
 endif
 
 ifdef SOUND_ARTS_SMOTEK
-CONFIG  += -DSYSDEP_DSP_ARTS_SMOTEK `artsc-config --cflags`
-LIBS += `artsc-config --libs`
+# CONFIG  += -DSYSDEP_DSP_ARTS_SMOTEK `artsc-config --cflags`
+# LIBS += `artsc-config --libs`
 endif
 
 ifdef SOUND_SDL
